### DISCUSSION TITLE
Compilation instructions

### DISCUSSION THREAD

**Discussion by vladdeSV** [jotaen/klog#70] [Help / Question] [answered]
I am very unfamiliar with Go. What is the procedure to compile *klog*?

Right now I want to use a binary which supports piping data. This should be fixed as of 633ac38dc9ae161d758e45e184fc1c7670d80311, and I want to use it.

(Maybe overkill, but is adding compilation instructions to the README something to consider?)

**Comment by jotaen** [comment, marked as answer]
I actually also thought about this today, ~will add it to the README.~ **UPDATE: Done**

In a nutshell, if you have Go 1.16 installed, you navigate to the `src` folder and run `go build app/cli/main/klog.go`. On first run that fetches all dependencies and then builds a `klog` binary. (On Windows you can add the `-o klog.exe` option for convenience.)

**Reply by jotaen** [reply]
https://github.com/jotaen/klog/issues/71
